resizer/resizehandleitem.h

```diff
@@ -37,6 +41,8 @@ class GraphicsItemResizer::HandleItem : public QGraphicsRectItem
 private:
     GraphicsItemResizer *resizer() const;
 
+    QRectF handlerRect(int attachment, const QSizeF &size) const;
```

Comment:
Probably it'd be more consistent if we name it `handleRect`
```suggestion
    QRectF handleRect(int attachment, const QSizeF &size) const;
```